### PR TITLE
Added an introduction and a prime-focus telescope tutorial to the documentation.

### PR DESCRIPTION
**Pull request by byrdie**

### CODE REVIEW

File: docs/tutorials/herschelian.ipynb
Comment (jacobdparker) [inline docs/tutorials/herschelian.ipynb:1]: I would think an example system plot and an example detector (possibly other surfaces as well) footprint plot are useful examples.  These do exist other places, like the Newtonian Telescope example.   Not sure how you want to organize them all.

File: docs/tutorials/herschelian.ipynb [hunk 1]
```diff
@@ -0,0 +1,622 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "650e2cb5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib notebook"
+   ]
+  },
+  {
```
Comment (jacobdparker) [inline docs/tutorials/herschelian.ipynb:553]: Do you want this decorator mid notebook?

**Comment by byrdie** [comment]
@wtbarnes, would you consider a short review about it reads from someone outside the project?

**Comment by wtbarnes** [comment]
Sorry, was late weighing in. I think the added info on the landing page and especially the tutorial notebook help to give a lot of context as to how to use this. Thanks for adding that!

**Comment by byrdie** [comment]
No, I'm sorry for not giving you more time to review. Let me know if you notice anything and I'll fix it with a quick PR.

**Comment by wtbarnes** [comment]
No worries and will do!

**Pull request merged**